logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/MLSConversationRepository.kt

```diff
@@ -126,6 +128,7 @@ interface MLSConversationRepository {
     suspend fun establishMLSGroup(
         groupID: GroupID,
         members: List<UserId>,
+        publicKeys: MLSPublicKeys? = null,
```

Comment:
question: there are places where this function is called with null and defaulting back to the server keys, mainly `JoinExistingMLSConversationUseCaseImpl`, `MLSMigrator` is this expected and what are the potential consciences of this, will it go back to the 50% chance of stuff being broken?

Reply:
The prekeys we get from the conversation response only needed when we want to create a conversation, there is no need to do so when we joining the existing conversations.